error-issue/src/issue_builder.ts

```diff
@@ -79,13 +83,34 @@ export class IssueBuilder {
     );
   }
 
-  possibleAssignees(): Array<string> {
-    // TODO(rcebulko): Identify reasonable assignees for the issue.
-    return [];
+  possibleAssignees(limit: number = 3): Array<string> {
+    const timeSinceError = Date.now() - this.firstSeen.valueOf();
+    if (timeSinceError > ONE_YEAR_MS) {
+      // Don't try to guess at assignees for old errors.
+      return [];
+    }
+
+    return (
+      this.blames
+        // Ignore large PRs like refactors and Prettier formatting.
+        .filter(({changedFiles}) => changedFiles <= MAX_CHANGED_FILES)
+        // Ignore PRs from before the issue first appeared.
+        .filter(({committedDate}) => committedDate < this.firstSeen)
+        // Ignore lines in the stacktrace from error throwing/logging.
+        .filter(({path}) => !ERROR_HANDLING_FILES.includes(path))
+        // Suggest most recent editors first.
+        .sort((a, b) => (a.committedDate < b.committedDate ? 1 : -1))
+        .map(({author}) => author)
+        .slice(0, limit)
+    );
   }
 
   get bodyNotes(): string {
-    const possibleAssignees = this.possibleAssignees()
+    if (!this.blames.length) {
+      return '';
+    }
+
+    const possibleAssignees = this.possibleAssignees(2)
```

Comment:
Why is the method's default value 3, and here you're calling it with 2? Would this realistically be changed by the caller? If not I think making it a const/env variable is a better approach

Reply:
Made it a const and made the const the default